server/php-master.cpp

```diff
@@ -1394,6 +1396,8 @@ static void cron() {
   CpuStatTimestamp cpu_timestamp{my_now, utime, stime, cpu_total};
   server_stats.update(cpu_timestamp);
 
+  vk::singleton<WebServerStats>::get().store({general_workers_stat.running_workers, general_workers_stat.waiting_workers,
```

Comment:
Are you sure, that here you modify shared memory?
As far as I can see, it's just a separate singleton, and I can't see the connection with `DataSharing` class, which is in shared memory.